eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpVariantsWriterTest.java

```diff
@@ -613,13 +641,32 @@ public void mergeThreeDuplicateClusteredVariantsInSameChunk() throws Exception {
 
         assertClusteredVariantStored(1, wrapper);
         assertSubmittedVariantsStored(1, submittedVariantEntity);
-        assertClusteredVariantMergeOperationStored(2);
+        assertClusteredVariantMergeOperationStored(2, clusteredVariantEntity);
         assertEquals(0, mongoTemplate.count(new Query(), DBSNP_CLUSTERED_VARIANT_DECLUSTERED_COLLECTION_NAME));
     }
 
-    private void assertClusteredVariantMergeOperationStored(int expectedOperations) {
+    private void assertClusteredVariantMergeOperationStored(int expectedOperations,
+                                                            DbsnpClusteredVariantEntity clusteredVariantEntity) {
         List<DbsnpClusteredVariantOperationEntity> operationEntities = mongoTemplate.find(
                 new Query(), DbsnpClusteredVariantOperationEntity.class);
         assertEquals(expectedOperations, operationEntities.size());
+        operationEntities
+                .stream()
+                .filter(op -> op.getMergedInto().equals(clusteredVariantEntity.getAccession()))
+                .forEach(operation -> {
+                    assertEquals(EventType.MERGED, operation.getEventType());
+                    List<DbsnpClusteredVariantInactiveEntity> inactiveObjects = operation.getInactiveObjects();
+                    assertEquals(1, inactiveObjects.size());
+                    assertNotEquals(clusteredVariantEntity.getAccession(), inactiveObjects.get(0).getAccession());
```

Comment:
I like how the first inactive object is extracted to a variable in the other class, but not a critical issue.